.github/workflows/test-e2e-gpu.yaml

```diff
@@ -11,6 +11,7 @@ jobs:
 
     env:
       GOPATH: ${{ github.workspace }}/go
+      HF_TOKEN: ${{ secrets.HF_TOKEN }}
```

Comment:
As we discussed, we also need to update the checkout action to ensure that it takes the latest changes from the PR:
```yaml
- uses: actions/checkout@v4
  with:
    ref: ${{ github.event.pull_request.head.sha }}
```